### DISCUSSION TITLE
How do i resolve this???

### DISCUSSION THREAD

**Discussion by David00154** [vercel/turborepo#4616] [Help] [answered]
![Screenshot from 2023-04-18 01-59-28](https://user-images.githubusercontent.com/56704811/232642309-2306c7f5-dab2-4d07-a31c-a1e74e20e161.png)
When I open the browser, it dosen't show anything but shows this error in the console.

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this problem, please open an issue with a reproduction. 🙏 Thank you!
